src/index.js

```diff
@@ -4,5 +4,12 @@ import './css/index.css';
 import './css/bootstrap.css';
 import App from './components/app';
 
+import { Provider } from 'react-redux';
+import store from './store';
 
-ReactDOM.render(<App />, document.getElementById('root'));
+
+ReactDOM.render(<Provider store={store}>
+		<App />
+	</Provider>, document.getElementById('root'));
+
+store.subscribe(() => console.log('New state', store.getState()));
```

Comment:
you already have logger in middleware. Do you need this line? It should be already log in your console.